pkg/providers/imagefamily/azlinux.go

```diff
@@ -80,17 +80,18 @@ func (u AzureLinux) DefaultImages() []DefaultImageOutput {
 func (u AzureLinux) ScriptlessCustomData(kubeletConfig *bootstrap.KubeletConfiguration, taints []v1.Taint, labels map[string]string, caBundle *string, _ *cloudprovider.InstanceType) bootstrap.Bootstrapper {
 	return bootstrap.AKS{
 		Options: bootstrap.Options{
-			ClusterName:      u.Options.ClusterName,
-			ClusterEndpoint:  u.Options.ClusterEndpoint,
-			KubeletConfig:    kubeletConfig,
-			Taints:           taints,
-			Labels:           labels,
-			CABundle:         caBundle,
-			GPUNode:          u.Options.GPUNode,
-			GPUDriverVersion: u.Options.GPUDriverVersion,
-			// GPUImageSHA: u.Options.GPUImageSHA - GPU image SHA only applies to Ubuntu
```

Comment:
More context on it applying only for Ubuntu: this refers to the container images published via aks-gpu (https://github.com/Azure/aks-gpu) that have the version and SHA suffix. 

Az Linux doesn't use those container images and instead installs the drivers during CSE. The version is not pinned per VHD/CSE - it just picks the latest. That is a major gap and needs to be fixed. 

Windows handles it separately.

Reply:
Also Az Linux does not support GRID GPUs today.

Reply:
The way I read it, _all_ CSE settings related to GPU driver image (type, version, sha) are currently ignored for Azure Linux, but it is should / is expected to start using them in the future. One option would be to remove setting all of them for Azure Linux (not just sha), but it seems we could just keep them all for symmetry with Ubuntu?